include/rosparam_shortcuts/rosparam_shortcuts.h

```diff
@@ -42,6 +42,10 @@
 // C++
 #include <string>
 #include <vector>
+#include <map>
+#include <mutex>
+#include <functional>
+#include <utility>
```

Comment:
Sort these alphabetically (maybe we should just find a clang-format rule to do this).  If you are using sublime text you can sort a block of lines by selecting them and pressing F9.